doc-content/enterprise-only/openshift/operator-createapp-proc.adoc

```diff
@@ -0,0 +1,125 @@
+** `rhpam-production-immutable`: An alternate environment for running existing services for staging and production purposes. This environment includes {CENTRAL} Monitoring and Smart Router; you can configure one or more {KIE_SERVER} replicated pods that build a service from source. In this environment, when you deploy a {KIE_SERVER} pod, it builds an image that loads and starts a service or group of services. You cannot stop any service on the pod or add any new service to the pod. If you want to use another version of a service or modify the configuration in any other way, you deploy a new server image and displace the old one. In this system, the {KIE_SERVER} runs like any other pod on the OpenShift environment; you can use any container-based integration workflows and do not need to use any other tools to manage the pods. 
+** `rhpam-authoring`: An environment for creating and modifying services using {CENTRAL}. It consists of pods that provide {CENTRAL} for the authoring work and a {KIE_SERVER} for test execution of the services.
+** `rhpam-authoring-ha`: An environment for creating and modifying services using {CENTRAL}. It consists of pods that provide {CENTRAL} for the authoring work and a {KIE_SERVER} for test execution of the services. This version of the authoring environment supports scaling the {CENTRAL} pod to ensure high availability.
++
+. Add lines to the YAML source to modify the environment:
+** The following snippet adds an immutable {KIE_SERVER} pod that builds a service from source. You must add at least one copy of this snippet when creating an immutable environment.
++
+[subs="attributes,verbatim,macros"]
+----
+  objects:
+    servers:
+      - build:
+          kieServerContainerDeployment: <deployment>
+          gitSource:
+            uri: <url>
+            reference: <branch>
+            contextDir: <directory>
+----
++
+Replace the following values:
++
+*** `<deployment>`: The identifying information of the decision service (KJAR file) that is built from your source. The format is: `<containerId>=<groupId>:<artifactId>:<version>`. You can provide two or more KJAR files using the `|` separator, for example: `containerId=groupId:artifactId:version|c2=g2:a2:v2`. The Maven build process must produce all these files from the source in the Git repository.
+*** `<url>`: The URL for the Git repository that contains the source for your decision service.
+*** `<branch>`: The branch in the Git repository.
+*** `<directory>`: The path to the source within the project downloaded from the Git repository (??? where is the artifact directory ???)
++
+** The following snippet sets the number of {KIE_SERVERS} that are managed by {CENTRAL} or {CENTRAL} Monitoring in your environment: 
++
+[subs="attributes,verbatim,macros"]
+----
+  objects:
+    server:
```

Comment:
We've consolidated the server/servers section into a single `servers` list. It's more intuitive and opens door for additional functionality. See -
https://github.com/kiegroup/kie-cloud-operator/blob/1.0.x/deploy/examples/server_config.yaml
https://github.com/kiegroup/kie-cloud-operator/blob/1.0.x/deploy/examples/immutable_s2i.yaml